crates/move-prover-boogie-backend/src/boogie_backend/bytecode_translator.rs

```diff
@@ -3569,6 +3571,13 @@ impl<'env> FunctionTranslator<'env> {
                         format!("{}({}, {})", update_fun, (*mk_dest)(), new_src)
                     }
                 }
+                BorrowEdge::EnumField(memory, _, vid) => {
```

Comment:
this should be handled the same way as `BorrowEdge::Field`, in order to support nested enums, or enum nested in structs, or structs nested in enums. This would require adding something like `boogie_enum_variant_field_sel` and `boogie_enum_variant_field_update`, similar to `boogie_field_sel` and `boogie_field_update `, and generating the respective boogie when generating the enums, if it's not already there.